invitations/base_invitation.py

```diff
@@ -1,6 +1,6 @@
 from django.conf import settings
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
No point in supporting this now so it can go

Reply:
Removed